apps/condo/domains/user/schema/AuthenticateUserWithPhoneAndPasswordService.js

```diff
@@ -13,8 +14,34 @@ const { RedisGuard } = require('@condo/domains/user/utils/serverSchema/guards')
 
 const redisGuard = new RedisGuard()
 
-const GUARD_WINDOW_SIZE_SEC = 60 * 60 // seconds
-const GUARD_WINDOW_LIMIT = 10
+const GUARD_DEFAULT_WINDOW_SIZE_SEC = 60 * 60 // seconds
+const GUARD_DEFAULT_WINDOW_LIMIT = 10
+
+/**
+ * @typedef {Object} TAuthGuardQuota
+ * @property {number} windowSizeSec The window size in seconds
```

Comment:
Same about `in` part
```suggestion
 * @property {number} windowSizeInSec The window size in seconds
```